contracts/abstract/ERC721H.sol

```diff
@@ -217,7 +217,7 @@ abstract contract ERC721H is Initializable {
    * @dev This function is unreachable unless custom contract address is called directly.
    *      Please use custom payable functions for accepting native value.
    */
-  receive() external payable {
+  receive() external payable virtual {
```

Comment:
is this so the custom contract inheriting can have a custom receive?

Reply:
yep, this is to allow custom contract to override it if needed